pkg/broker/handler/processors/deliver/processor_test.go

```diff
@@ -272,6 +272,63 @@ func (h *statusCodeReplyHandler) ServeHTTP(w http.ResponseWriter, req *http.Requ
 	}
 }
 
+func TestProcess_TriggerWithoutSubscriberAddress(t *testing.T) {
```

Comment:
Should we also test the empty address for other CellTenant type (channel) to make sure that it will be delivered to the specified reply address? Also, what if the reply address is empty too?

Nits: we are not using "_" in test names.

Reply:
We should have a test that shows that. Unfortunately, I can't at the moment. There are only two values for the CellTenantType enum, BROKER and UNKNOWN. We have defaulting logic that pretends UNKNOWNs are actually BROKER. So for now, I can't add this test.

However, I set up the test to make it possible and included a commented out version of what the test will be.